pkg/config/config.go

```diff
@@ -49,3 +50,38 @@ func GetDefaultConfigPath() (string, error) {
 
 	return filepath.Join(home, DefaultConfigPath, DefaultConfigFileName), nil
 }
+
+func GetPathToCalculation(defaultPath string, fileName string) (string, error) {
+	user, err := user.Current()
+	if err != nil {
+		return "", fmt.Errorf("couldn't get user's information: %v", err)
+	}
+	home := user.HomeDir
+
+	if defaultPath == "" {
+		return filepath.Join(home, DefaultConfigPath, fileName), nil
+	}
+
+	_, err = os.Stat(defaultPath)
+	if err != nil {
+		return "", fmt.Errorf("something is wrong with the download path of the calculation %v", defaultPath)
```

Comment:
There is no reason to eat the actual error message with a `something is wrong` message. We need the user to know exactly why his actions failed. We need something like:
```suggestion
		return "", fmt.Errorf("couldn't stat path %s: %v", defaultPath, err)
```